packages/cli/lib/strings.js

```diff
@@ -0,0 +1,22 @@
+const chalk = require('chalk');
+
+const PROJECTS = {
+  BUILD: {
+    INITIALIZE: (name, numOfComponents) =>
+      `Building ${chalk.bold(
+        name
+      )}\n\nFound ${numOfComponents} components in this project ...\n`,
+    SUCCESS: name => `Built ${chalk.bold(name)}`,
+  },
+  DEPLOY: {
+    INITIALIZE: (name, numOfComponents) =>
+      `Deploying ${chalk.bold(
+        name
+      )}\n\nFound ${numOfComponents} components in this project ...\n`,
+    SUCCESS: name => `Deployed ${chalk.bold(name)}`,
+  },
+};
+
+module.exports = {
+  PROJECTS,
+};
```

Comment:
@brandenrodgers @miketalley I wanted to centralize some strings to make upkeep easier. Before I go too much further does this seem too weird? Open to suggestions!